packages/jspsych/src/ProgressBar.ts

```diff
@@ -14,7 +17,13 @@ export class ProgressBar {
   /** Adds the progress bar HTML code into `this.containerElement` */
   private setupElements() {
     this.messageSpan = document.createElement("span");
-    this.messageSpan.innerHTML = this.message;
+
+    if (Object.prototype.toString.call(this.message) === "[object Function]") {
```

Comment:
Why not use `typeof this.message === "function"`?

Reply:
My mistake. I think I mixed it up with something else.